client/web/compose/src/components/PageBlocks/Navigation/NavTypes/Compose.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <tr>
+    <td />
+    <td>
+      <b-form-group :label="$t('navigation.fieldLabel')">
+        <b-form-input
+          v-model="value.options.itemOption.label"
+          type="text"
+          size="sm"
+        />
+      </b-form-group>
+    </td>
+    <td>
+      <b-form-group :label="$t('navigation.composePage')">
+        <b-form-select
+          v-model="value.options.itemOption.referenceId"
+          :options="composePageList"
+          size="sm"
+          @change="updateLabelValue"
+        >
+          <template #first>
+            <b-form-select-option
+              :value="undefined"
+            >
+              {{ $t('navigation.none') }}
+            </b-form-select-option>
+          </template>
+        </b-form-select>
+      </b-form-group>
+    </td>
+    <td>
+      <b-form-group :label="$t('navigation.openIn')">
+        <b-form-select
+          v-model="value.options.itemOption.targetType"
+          :options="openInType"
+          size="sm"
+        />
+      </b-form-group>
+    </td>
+    <td
+      v-if="selectedPageChildren(value.options.itemOption.referenceId)"
+      cols="12"
+      sm="6"
+      class="align-middle text-center"
+    >
+      <b-form-group class="m-0">
+        <b-form-checkbox
+          v-model="value.options.itemOption.displaySubPages"
+          name="enabled-button"
+          switch
+          class="text-left"
+        >
+          {{ $t('navigation.showReferencedPage') }}
+        </b-form-checkbox>
+      </b-form-group>
+    </td>
+    <td />
+  </tr>
+</template>
+
+<script>
+import { compose } from '@cortezaproject/corteza-js'
+
+export default {
+  i18nOptions: {
+    namespaces: 'block',
+  },
+
+  props: {
+    column: {
+      type: Object,
+      required: true,
+    },
+
+    namespace: {
+      type: compose.Namespace,
+      required: true,
+    },
+  },
+
+  data () {
+    return {
+      tree: [],
+      openInType: [
+        { value: '_self', text: this.$t('navigation.sameTab') },
+        { value: '_blank', text: this.$t('navigation.newTab') },
```

Comment:
Since the two values are the same, if I save new window it thinks I saved newTab. So you'll need something like Atanas did in record-list for the `recordDisplayOption`

Reply:
the value for new window and new tab is the same for the target value, I don't seem to know why. Atanas implementation does not have a new window